info.json

```diff
@@ -107,36 +102,36 @@
             }
         ],
         "globalVariables": [
+            {
+                "id": 16,
+                "name": "GDPR_Demo_Email",
+                "value": "rktest.2204@outlook.com",
+                "default_value": "no-reply@example.com"
+            },
             {
                 "id": 15,
                 "name": "GDPR_Demo_Mode",
                 "value": "Demo Mode ON",
```

Comment:
value should be ON/OFF or TRUE/FALSE or YES/NO

Reply:
'GDPR_Demo_Mode' global variable is now removed